book/src/tooling-check.md

```diff
@@ -30,6 +30,52 @@ Hello, world!
 
 This ensures that the whole toolchain works correctly and finds the system linker. This should work at all times, if it doesn't, immediately ask for a trainer.
 
-# QT specifics
+# Qt specifics
 
-TODO tool check for QT, compiling a hello world app.
+For those interested in following the Qt section of the course,
```

Comment:
this is all a slight repeat of what is in `book/src/setup.md` but makes sense i guess as it's important